randomize/plugin.py

```diff
@@ -19,8 +19,37 @@
 from nose.util import isclass, isgenerator, transplant_func, transplant_class
 import random
 import unittest
+from random import getrandbits
```

Comment:
Could we remove this and just call random.getrandbits()? That's sort of the convention being used in this module